ssl/record/ssl3_buffer.c

```diff
@@ -160,7 +163,9 @@ int ssl3_release_write_buffer(SSL *s)
     while (pipes > 0) {
         wb = &RECORD_LAYER_get_wbuf(&s->rlayer)[pipes - 1];
 
-        OPENSSL_free(wb->buf);
+        if (wb->buf)
```

Comment:
There is no need for this test. `OPENSSL_free` can cope with a NULL input

Reply:
Done